ttim/aquifer.py

```diff
@@ -58,6 +58,9 @@ def __init__(
         self.model3d = model3d
         if self.model3d:
             assert self.kzoverkh is not None, "model3d specified without kzoverkh"
+            assert (
+                self.topboundary != "con"
+            ), "Error: For Model3D, only 'confined' topboundary is implemented yet"
```

Comment:
remove the "yet", now the sentence is a bit awkward.

Reply:
Ah, yes, line lengths :). You could split the string: 

```python
assert self.topboundary != "con", (
    "Error: For Model3D, only 'confined' topboundary is "
    "currently implemented because of reason reason reason reason"
)
```